data/pavanh007

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Pavan H",
+    "type": "personal",
+    "bio": "Backend developer" | "Tech enthusiast",
```

Comment:
The format is incorrect. The correct format is "Backend developer | Tech enthusiast"